examples/diagram-fit-variants/index.html

```diff
@@ -179,31 +179,31 @@ <h2>Various fit variants</h2>
                     break;
             }
 
-            bpmnVisualization.graph.fit(fitBorder, false, 0, true, ignoreWidth, ignoreHeight);
+            bpmnVisualization.graph.fit(fitMargin, false, 0, true, ignoreWidth, ignoreHeight);
         }
 
-        // 'Fit Border' dropdown list
-        const fitBorderValues = [0, 5, 10, 20, 50];
-        const dropdownFitBorderElt = document.querySelector('#dropdown-fit-border');
-        dropdownFitBorderElt.innerHTML = fitBorderValues.map(fitBorder => {
-            return `<li><a href="#" class="btn btn-link">${fitBorder}</a></li>`
+        // 'Fit Margin' dropdown list
+        const fitMarginValues = [0, 5, 10, 20, 50];
+        const dropdownFitMarginElt = document.querySelector('#dropdown-fit-margin');
```

Comment:
I like the suffix Elt in variableNameElt to point it is an Element
I think it should be our convention 👍